app/pages/project/instances/instance/SerialConsolePage.tsx

```diff
@@ -20,13 +20,12 @@ import { PrevArrow12Icon } from '@oxide/design-system/icons/react'
 
 import { EquivalentCliCommand } from '~/components/CopyCode'
 import { InstanceStateBadge } from '~/components/StateBadge'
+import { Terminal } from '~/components/Terminal'
 import { getInstanceSelector, useInstanceSelector } from '~/hooks/use-params'
 import { Badge, type BadgeColor } from '~/ui/lib/Badge'
 import { Spinner } from '~/ui/lib/Spinner'
 import { pb } from '~/util/path-builder'
 
-const Terminal = lazy(() => import('~/components/Terminal'))
-
```

Comment:
Nice example of what is happening here: we get this for free through the route being split.